alternator/executor.cc

```diff
@@ -3221,6 +3222,9 @@ future<executor::request_return_type> executor::batch_write_item(client_state& c
                 w.first->wcu_total[stats::DELETE_ITEM] += wcu;
                 wcu_delete_units += wcu;
             }
+            if (item_size) {
```

Comment:
How can item_size be zero? How? Doesn't it always include the length of the key?